src/assets/data/edition/series/1/section/5/op3/source-description.json

```diff
@@ -3151,7 +3151,7 @@
                                         "measure": "9",
                                         "system": "Klav. o.",
                                         "position": "1.–5. Note",
-                                        "comment": "sic: Bogen. Siehe Korrekturen 2 in <strong>F<sup>b</sup></strong>."
+                                        "comment": "sic: Bogen. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op3', fragmentId: 'source_Fb_corr2'})\">Korrekturen 2 in <strong>F<sup>b</sup></strong></a>."
```

Comment:
Nach bisheriger Logik hier "Korrekturen 2" ausreichend (ohne in Fb). Oder sonst der Präzision halber ergänzen?